mackerel-plugin-linux/lib/linux_test.go

```diff
@@ -167,3 +142,28 @@ pswpout 113`
 	assert.EqualValues(t, stat["pswpin"], 0)
 	assert.EqualValues(t, stat["pswpout"], 113)
 }
+
+func TestCollectDiskstats(t *testing.T) {
```

Comment:
The method name is `collectDiskStats`, so `TestCollectDiskStats` is better than `TestCollectDiskstats`?